apis/config/v1beta1/configuration_types.go

```diff
@@ -192,8 +192,22 @@ type WaitForPodsReady struct {
 	// until the jobs reach the PodsReady=true condition. It defaults to false if Enable is false
 	// and defaults to true otherwise.
 	BlockAdmission *bool `json:"blockAdmission,omitempty"`
+
+	// RequeuingTimestamp defines the timestamp used for requeuing a Workload
+	// that was evicted due to Pod readiness. Defaults to Eviction (back of queue).
+	RequeuingTimestamp RequeuingTimestamp `json:"requeuingTimestamp,omitempty"`
```

Comment:
As this is a new field I think it should be a ptr, and only use Ordering, Eviction, or null (defaulted to Eviction).
I think otherwise the API clients generated from the new code may crash when dealing with old API, because they may expect the field to be always returned.

Reply:
This is still unresolved

Reply:
I can see it resolved now, feel free to resolve the comment